pcdsdevices/smarpod.py

```diff
@@ -0,0 +1,266 @@
+    pz_m = Cpt(EpicsSignal, ':PZ_M', kind='normal')
+    px = Cpt(EpicsSignal, ':PX', kind='normal')
+    py = Cpt(EpicsSignal, ':PY', kind='normal')
+    pz = Cpt(EpicsSignal, ':PZ', kind='normal')
+
+
+
+    """
+    Pivot request positions
+    """
+    cmd_px = Cpt(EpicsSignal, ':CMD:PX', kind='normal')
+    cmd_py = Cpt(EpicsSignal, ':CMD:PY', kind='normal')
+    cmd_pz = Cpt(EpicsSignal, ':CMD:PZ', kind='normal')
+
+
+
+    """
+    Miscellaneous
+    """
+    cmd_calibrate = Cpt(EpicsSignal, ':CMD:CALIBRATE', kind='normal')
+    cmd_read_ver = Cpt(EpicsSignal, ':CMD:READ_VER', kind='normal')
+    ver_sys = Cpt(EpicsSignal, ':VER:SYS', kind='normal')
+
+
+
+    """
+    Device Info
+    """
+    ver_sn  = Cpt(EpicsSignal, ':VER:SN', kind='normal')
+    ver_product  = Cpt(EpicsSignal, ':VER:PRODUCT', kind='normal')
+    ver_firmware = Cpt(EpicsSignal, ':VER:FIRMWARE', kind='normal')
+
+
+
+    """
+    Status
+    """
+    cmd_read_status = Cpt(EpicsSignal, ':CMD:READ_STATUS', kind='normal')
+    status_temp = Cpt(EpicsSignal, ':STATUS:TEMP', kind='normal')
+    status_load = Cpt(EpicsSignal, ':STATUS:LOAD', kind='normal')
+    status_memory = Cpt(EpicsSignal, ':STATUS:MEMORY', kind='normal')
+    status_net_bytes_in = Cpt(EpicsSignal, ':STATUS:NET_BYTES_IN', kind='normal')
+    status_net_bytes_out = Cpt(EpicsSignal, ':STATUS:NET_BYTES_OUT', kind='normal')
+    status_net = Cpt(EpicsSignal, ':STATUS:NET', kind='normal')
+    status_status = Cpt(EpicsSignal, ':STATUS:STATUS', kind='normal')
+    status_bootcnt = Cpt(EpicsSignal, ':STATUS:BOOTCNT', kind='normal')
+    status_uptime = Cpt(EpicsSignal, ':STATUS:UPTIME', kind='normal')
+    status_ip = Cpt(EpicsSignal, ':STATUS:IP', kind='normal')
+    status_client_ip = Cpt(EpicsSignal, ':STATUS:CLIENT_IP', kind='normal')
+    status_ser_status = Cpt(EpicsSignal, ':STATUS:SER_STATUS', kind='normal')
+    status_ser_bytes_in = Cpt(EpicsSignal, ':STATUS:SER_BYTES_IN', kind='normal')
+    status_ser_bytes_out = Cpt(EpicsSignal, ':STATUS:SER_BYTES_OUT', kind='normal')
+    status_display = Cpt(EpicsSignal, ':STATUS:DISPLAY', kind='normal')
+ 
+
+
+    """
+    Sensor mode
+    """
+    cmd_sensor_mode = Cpt(EpicsSignal, ':CMD:SENSOR_MODE', kind='normal')
+    sensor_mode = Cpt(EpicsSignal, ':SENSOR_MODE', kind='normal', doc='Sensor mode')
+
+
+
+    """
+    Maximum frequency 
+    """
+    cmd_freq = Cpt(EpicsSignal, ':CMD:FREQ', kind='normal')
+    freq = Cpt(EpicsSignal, ':FREQ', kind='normal')
+
+
+
+    """
+    Maximum velocity
+    """
+    cmd_vel = Cpt(EpicsSignal, ':CMD:VEL', kind='normal')
+    vel = Cpt(EpicsSignal, ':VEL', kind='normal')
+
+
+
+    """
+    Maximum acceleration
+    """
+    cmd_accel = Cpt(EpicsSignal, ':CMD:ACCEL', kind='normal')
+    accel = Cpt(EpicsSignal, ':ACCEL', kind='normal')
+
+
+
+    """
+    Read and Set Poses, Pose 1-5:
+    """
+    pose_1_name = Cpt(EpicsSignal, ':POSE_1:NAME', kind='normal', doc='Pose 1 name input')
```

Comment:
Any reason not to make a single pose a sub-device?

Reply:
It was incorporated according to Ken's suggestion.